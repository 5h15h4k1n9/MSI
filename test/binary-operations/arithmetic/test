1 + 2
5 - 3
4 * 7
20 / 4
17 % 5
exit
