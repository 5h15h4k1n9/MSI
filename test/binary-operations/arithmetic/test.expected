(int) 3
(int) 2
(int) 28
(int) 5
(int) 2
Bye-bye!
